package.json

```diff
@@ -45,7 +52,7 @@
     "prepare": "husky install && patch-package",
     "release": "np --no-release-draft",
     "pretest": "npm run lint",
```

Comment:
question

Shouldn't `npm run build` be in pretest instead?
That way `test-only` won't build.

Reply:
`pretest` isn't run since we've specified `--ignore-scripts` to split `test` and `lint` jobs:

https://github.com/stylelint/.github/blob/8091030182f57ca1704710cfb926d9be45eb4a0e/.github/workflows/test.yml#L69

So I've just removed `npm run build` from `test` via c9a99a30.